easybuild/easyconfigs/d/Datasets/Datasets-2.18.0-gfbf-2023a.eb

```
@@ -0,0 +1,48 @@
+easyblock = "PythonBundle"
+
+name = 'Datasets'
```

Perhaps the name is a bit to generic. "HF-Datasets"?

---

```suggestion
name = 'HF-Datasets'
```
That or HuggingFace-Datasets (though, same could be said for Transformers (though that is a bit less generic at least))